src/lockfile/wrapper.js

```diff
@@ -86,9 +87,10 @@ export default class Lockfile {
     [key: string]: LockManifest,
   };
 
-  static async fromDirectory(dir: string, reporter?: Reporter): Promise<Lockfile> {
+  static async fromDirectory(config: Config, dir: string, reporter?: Reporter): Promise<Lockfile> {
     // read the manifest in this directory
-    const lockfileLoc = path.join(dir, constants.LOCKFILE_FILENAME);
+    const lockfileLoc = path.join(config.worktreeFolder || config.cwd, constants.LOCKFILE_FILENAME);
```

Comment:
Good catch! This code became boggus after I refactored it a few times. I've now restored the old code, since we don't use `config` in this function anymore 👍